lib/reek/report/formatter.rb

```diff
@@ -42,6 +43,11 @@ def format_hash(warning)
         warning.yaml_hash
       end
 
+      # :reek:UtilityFunction
+      def format_code_climate_hash(warning)
+        CodeClimateFormatter.new(warning).to_hash
+      end
```

Comment:
Since we already have `format_hash` above, I would keep it like this for now and refactor both later.

On November 7, 2015 9:21:07 PM GMT+01:00, Andy Waite <notifications@github.com> wrote:
>> @@ -42,6 +43,11 @@ def format_hash(warning)
>>          warning.yaml_hash
>>        end
>>  
>> +      # :reek:UtilityFunction
>> +      def format_code_climate_hash(warning)
>> +        CodeClimateFormatter.new(warning).to_hash
>> +      end
>
>Yeah, I considered that, but thought it might be better to keep it
>consistent with the existing JSON formatter. WDYT @troessner? I do
>prefer it the way @chastell suggests.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/troessner/reek/pull/763/files#r44216719

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.